app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -117,7 +129,8 @@ private fun HomeScreenContent(
                         Text(
                             modifier = Modifier
                                 .fillMaxWidth()
-                                .padding(top = Dp16),
+                                .padding(top = Dp16)
+                                .testTag(tag = stringResource(id = R.string.home_title)),
```

Comment:
I think we should have a const for this tag, instead of using a string

Reply:
I agree as we have created `const for tag` 🚀